.github/workflows/ci.yml

```
@@ -6,7 +6,7 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
```

```suggestion
      fail-fast: true
```

this should maybe also be reverted? 

(for working on ci, the setting is a bit annoying though)

---

Thanks, yeah probably best to revert that.